contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/gst_resources_generator.py

```diff
@@ -202,7 +201,7 @@ def gst_v4l2_audio_video_synchronization(
                 )
 
     def gst_v4l2_video_decoder_performance_fakesink(
-        self, scenario_data: List[Dict]
+        self, scenario_data: list[dict]
```

Comment:
Using `list` `dict` in type annotations is not supported in python 3.5 (not an issue if we can guarantee python > 3.9

Reply:
Thanks your catch

Reply:
Since we are still having the requirement of Ubuntu 20.04 (python3.8), I think we have to use Dict and List.

Reply:
[ae97ce7](https://github.com/canonical/checkbox/pull/1516/commits/ae97ce7bbfac3176aaeba5efefc4eb78967dc175)